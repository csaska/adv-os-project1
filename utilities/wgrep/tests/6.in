version https://git-lfs.github.com/spec/v1
oid sha256:be8565072f5aca95cf28693d92f12c6ffa158d7d61effa035f889c5c21ef72c9
size 22864
