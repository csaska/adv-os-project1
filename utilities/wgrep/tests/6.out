version https://git-lfs.github.com/spec/v1
oid sha256:444caa3fc469fd7037580a2462ee26ee7c0739f4b25f8937c37e19182b4e98a6
size 22808
